gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+  * Copyright 2014 Netflix, Inc.
```

Comment:
you work at netflix? :D

Reply:
Copy pasted :D now I guess I do.

Reply:
Removing it altogether - unless we have our own correct/legal one...

Reply:
Just replace `Netflix` with `Lookout`